symphony.rb

```diff
@@ -0,0 +1,74 @@
+class Symphony < Formula
+  desc "Framework for solving mixed integer linear programs"
+  homepage "https://projects.coin-or.org/SYMPHONY"
+  url "http://www.coin-or.org/download/pkgsource/SYMPHONY/SYMPHONY-5.6.14.tgz"
+  sha256 "666d1c72c4bb084b470c4f066b1694f7b3b11a479de15d9d1e9bfe1b53e3ae63"
+  head "https://projects.coin-or.org/svn/SYMPHONY/trunk"
+
+  option "without-openmp", "Disable openmp support"
```

Comment:
OK, I added this now, but then do I need to list an explicit dependency on gcc? And then do all projects have to be built with clang. Right now, I'm getting a strange link error around [here](https://travis-ci.org/coin-or-tools/homebrew-coinor/jobs/136560954#L732). I can't really see what's wrong. The link line looks fine. I can't test locally because my homebrew gcc seems to be broken in a strange way so I'm trying to upgrade. 

It did occur to me that OSI might be listed as a direct dependency, since it is needed to build `OsiSymSolverInterface`. I didn't list it since `clp` is a dependency and `clp` depends on Osi.